here-naksha-app-service/src/test/java/com/here/naksha/app/service/DefaultViewHandlerTest.java

```diff
@@ -0,0 +1,364 @@
+                .hasJsonBody(expectedBodyPart, "Create Feature response body doesn't match");
+    }
+
+    @Test
+    void tc5020_searchByBBox_AvailableInAllSpaces() throws Exception {
+        //given Feature with this id is available in all spaces (delta,dlb,base)
+        final String bboxQueryParam = "west=18.5&south=54.5&east=19&north=55";
+
+        final String expectedBodyPart = loadFileOrFail("ViewHandler/TC5020_searchByBBox/feature_response_part.json");
+        String streamId = UUID.randomUUID().toString();
+
+        // When: Get Features By BBox request is submitted to NakshaHub
+        HttpResponse<String> viewResponse = nakshaClient
+                .get("hub/spaces/" + SPACE_ID + "/bbox?" + bboxQueryParam, streamId);
+
+        //then expect that feature from last layer will be returned (delta)
+        assertThat(viewResponse)
+                .hasStatus(200)
+                .hasStreamIdHeader(streamId)
+                .hasJsonBody(expectedBodyPart, "Create Feature response body doesn't match");
+    }
+
+    @Test
+    void tc5021_searchByBBox_AvailableInDlbAndBase() throws Exception {
+        //given Feature with this id is available in base and dlb
+        final String bboxQueryParam = "west=5.6&south=40.0&east=5.7&north=40.1";
+
+        final String expectedBodyPart = loadFileOrFail("ViewHandler/TC5021_searchByBBox/feature_response_part.json");
+        String streamId = UUID.randomUUID().toString();
+
+        // When: Get Features By BBox request is submitted to NakshaHub
+        HttpResponse<String> viewResponse = nakshaClient
+                .get("hub/spaces/" + SPACE_ID + "/bbox?" + bboxQueryParam, streamId);
+
+        //then expect that feature from last layer will be returned (dlb)
+        assertThat(viewResponse)
+                .hasStatus(200)
+                .hasStreamIdHeader(streamId)
+                .hasJsonBody(expectedBodyPart, "Create Feature response body doesn't match");
+    }
+
+    @Test
+    void tc5021_searchByBBox_AvailableOnlyInBase() throws Exception {
+        //given Feature with this id is available only in base
+        final String bboxQueryParam = "west=0.5&south=30.0&east=1&north=30.1";
+
+        final String expectedBodyPart = loadFileOrFail("ViewHandler/TC5022_searchByBBox/feature_response_part.json");
+        String streamId = UUID.randomUUID().toString();
+
+        // When: Get Features By BBox request is submitted to NakshaHub
+        HttpResponse<String> viewResponse = nakshaClient
+                .get("hub/spaces/" + SPACE_ID + "/bbox?" + bboxQueryParam, streamId);
+
+        //then expect that feature from last layer will be returned (dlb)
+        assertThat(viewResponse)
+                .hasStatus(200)
+                .hasStreamIdHeader(streamId)
+                .hasJsonBody(expectedBodyPart, "Create Feature response body doesn't match");
+    }
+
+    @Test
+    void tc5021_searchByBBox_PullingByIdWhenOutsideBBox() throws Exception {
```

Comment:
This test "tc5021_searchByBBox_PullingByIdWhenOutsideBBox" seems to be duplicate of above one. Can this be safely removed?